src/Uno.UI/UI/Xaml/Automation/AnnotationPatternIdentifiers.cs

```diff
@@ -0,0 +1,36 @@
+namespace Windows.UI.Xaml.Automation;
+
+/// <summary>
+/// Contains values used as identifiers by IAnnotationProvider.
+/// </summary>
+public partial class AnnotationPatternIdentifiers
+{
+	internal AnnotationPatternIdentifiers()
+	{
+	}
+
+	/// <summary>
+	/// Gets the identifier for the AnnotationTypeId automation property.
+	/// </summary>
+	public static AutomationProperty AnnotationTypeIdProperty => new();
```

Comment:
I would be surprised if WinUI returns a new instance each time

Reply:
Yes, should be `{ get; } = new();`